Foundation/NSThread.swift

```diff
@@ -57,18 +57,30 @@ private func NSThreadStart(_ context: UnsafeMutableRawPointer?) -> UnsafeMutable
 }
 
 open class Thread : NSObject {
-    
+    static internal let _mainThread = Thread(thread: pthread_self())
```

Comment:
When is this initializer actually called? Is it lazy, so the first to ask for it will be implicitly main?